VirtusLab-Open-Source/strapi-plugin-navigation
# feat: media additional field

## Ticket

https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/issues/337

## Summary

What does this PR do/solve? 

- `media` as available additional navigation item type

## Test Plan

- start the app
- in settings view add custom field with a `media` type
- go to navigation details view
- add or update navigation item to set an image in a custom field
- save navigation
- render updated navigation through an API call
- custom field media details should be available
